protocol/communities/community.go

```diff
@@ -1547,6 +1547,10 @@ func (o *Community) HasTokenPermissions() bool {
 	return len(o.tokenPermissions()) > 0
 }
 
+func (o *Community) ChannelEncrypted(channelID string) bool {
+	return o.ChannelHasTokenPermissions(o.IDString() + channelID)
```

Comment:
Maybe we should create a function to get the channel ID. I think we have this concatenation in multiple places

Reply:
:+1: extracted